test/tests/WriteTest.cpp

```diff
@@ -18,14 +18,15 @@ struct WriteTest : tpunit::TestFixture {
                               TEST(WriteTest::failedUpdateNoWhereFalse),
                               TEST(WriteTest::updateAndInsertWithHttp),
                               TEST(WriteTest::shortHandSyntax),
+                              TEST(WriteTest::keywordsAsValue),
                               AFTER_CLASS(WriteTest::tearDown)) { }
 
     BedrockTester* tester;
 
     void setup() {
         tester = new BedrockTester({}, {
             "CREATE TABLE foo (bar INTEGER);",
-            "CREATE TABLE stuff (id INTEGER PRIMARY KEY, value INTEGER);",
+            "CREATE TABLE stuff (id INTEGER PRIMARY KEY, value INTEGER, info TEXT);",
```

Comment:
curious why this change and the one below?

Reply:
We want to include a test where SQL keywords like UPDATE, DELETE are inserted into the table as a values. There was not any column to add TEXT data, so created a column here.
Below in the other query, passed NULL to match the table schema.